src/App.js

```diff
@@ -7,23 +7,10 @@ import { Navigate, Route, Routes } from 'react-router-dom';
 import Navbar from './components/Navbar';
 import { AuthStatus, useAuth } from '@akosasante/react-auth-context';
 
```

Comment:
We can also remove `useEffect` and `getAllData` from the imports as they're not used anymore

Reply:
I removed them